spinegeneric/cli/generate_figure.py

```diff
@@ -527,7 +526,7 @@ def format_p_value(p_val):
 
         # Get significant post-hoc results
         index = sum(
-            stats["tukey_test"].reject == True
+            bool(stats["tukey_test"].reject)
```

Comment:
I have second-thoughts about this line. `flake8` flags any `== True` as a mistake, but this is doing a vectorized comparison. In fact you shouldn't be able to call `sum(bool(_))` because the types are mismatched: `sum()` takes a list.

I would have to take some time to learn how to/write a test for this line before this can be merged.

Reply:
I think pandas (and numpy) get a special pass for the flake8 lints about comparison operators, since they have unexpected (to flake8) semantics. I reinstated the `== True` comparison and added a `noqa: 712` comment to silence the warning.